telegraf/README.md

```diff
@@ -1,9 +1,6 @@
 # Telegraf
 [![Docker Repository on Quay](https://quay.io/repository/deisci/telegraf/status "Docker Repository on Quay")](https://quay.io/repository/deisci/telegraf)
 
-## Description
```

Comment:
Should we just update this description to better match what telegraf does for us?